la.ogri.strongbox.yml

```diff
@@ -0,0 +1,43 @@
+app-id: la.ogri.strongbox
```

Comment:
ogri.la does not resolve see https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
> The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.

```bash
$ dig -t NS +short ogri.la
ns-1269.awsdns-30.org.
ns-1939.awsdns-50.co.uk.
ns-433.awsdns-54.com.
ns-790.awsdns-34.net.
```

what is the domain expected to resolve to? I can add a dns record to prove I have control over it.

Reply:
usually we check with the presence of the website, because
1. it's used for the opt-in verification
2. anything else is just impossible.

Reply:
`ogri.la` now has a website:

```bash
$ curl --head --http1.1 http://ogri.la
HTTP/1.1 200 OK
x-amz-id-2: BQXfs/nFP+yM3RleNnpcPsES3lLLlyJd2IzBSmBi4KPzCACGdob/0kE534k7F8rTvIQA5HTYDoc=
x-amz-request-id: YKHHGPP8M6841SA0
Date: Sun, 12 Nov 2023 01:25:44 GMT
Last-Modified: Thu, 09 Nov 2023 06:19:41 GMT
ETag: "1831ed11c59d24312a7fcc161bb34d9c"
Content-Type: text/html
Server: AmazonS3
Content-Length: 235
```